src/anshuKJun20/AppendLastString.java

```diff
@@ -0,0 +1,24 @@
+package anshuKJun20;
+
+public class AppendLastString {
+	
+	void appendLastString(String input ,String requiredOutput) {
+		int count =0;
+		if(input.length() == requiredOutput.length()) {
```

Comment:
you should add else part too. if it is not equal tell the user it cant be made identical.